docs/resources/Guild.md

```diff
@@ -283,6 +283,7 @@ A partial [guild](#DOCS_RESOURCES_GUILD/guild-object) object. Represents an Offl
 | mute           | boolean                                         | whether the user is muted in voice channels                                                                                            |
 | pending?       | boolean                                         | whether the user has not yet passed the guild's [Membership Screening](#DOCS_RESOURCES_GUILD/membership-screening-object) requirements |
 | permissions?   | string                                          | total permissions of the member in the channel, including overwrites, returned when in the interaction object                           |
+| avatar?        | string                                          | the member's [guild avatar hash](#DOCS_REFERENCE/image-formatting)
```

Comment:
Seems weird how avatar isn't nullable here but it is in the user structure 🤔

Reply:
Seems like I didn't look at the member data correctly, fixed.

Reply:
Lol no worries